components/html-block/demo/html-block.html

```diff
@@ -54,28 +54,26 @@ <h2>HTML Block</h2>
 			<d2l-demo-snippet>
 				<template>
 					<d2l-html-block>
-						<template>
-							<h1>heading 1</h1>
-							<h2>heading 2</h2>
-							<h3>heading 3</h3>
-							<h4>heading 4</h4>
-							<h5>heading 5</h5>
-							<h6>heading 6</h6>
-							<div><strong>strong</strong></div>
-							<div><b>bold</b></div>
-							<div>text</div>
-							<pre>preformatted</pre>
-							<p>paragraph</p>
-							<ul>
-								<li>unordered item 1</li>
-								<li>unordered item 2</li>
-							</ul>
-							<ol>
-								<li>ordered item 1</li>
-								<li>ordered item 2</li>
-							</ol>
-							<div><a href="https://d2l.com">anchor</a></div>
-						</template>
+						<h1>heading 1</h1>
```

Comment:
Same as for visual-diff, I am thinking it would be good to include a text node as a direct child of `d2l-html-block` as well.